velox/docs/monthly-updates/april-2023.rst

```diff
@@ -0,0 +1,75 @@
+********************
+April 2023 Update
+********************
+
+
+Documentation
+=============
+
+ * Add :doc:`documentation on UnsafeRow serialization <../develop/unsaferow>`.
+
+
+Core Library
+============
+
+ * Add support for peeling past a loaded lazy layer.
+ * Add support for companion function generation for aggregations.
+ * Add support to suppress errors for unused arguments of null-propagating functions.
+
+
+Presto Functions
+================
+
+ * Add :func:`timezone_minute`,   :func:`timezone_hour` functions.
+ * Add :func:`date`,   :func:`random()`,  :func:`sequence`,  :func:`from_base64url` functions.
+ * Add support for DECIMAL comparison functions.
+ * Add support for DECIMAL :func:`abs` and :func:`negate` functions.
+ * Add :func:`first_value` and :func:`last_value` window functions.
+ * Add :func:`from_big_endian_32`,  :func:`from_big_endian_64`,  :func:`to_big_endian_32`,  :func:`to_big_endian_64` functions.
```

Comment:
Move this item next to other functions. Make sure functions are grouped according to some pattern.